<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wndRun</class>
 <widget class="QMainWindow" name="wndRun">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1033</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>eudaq Run Control</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="QGroupBox" name="grpCurrentState">
      <property name="title">
       <string>State:</string>
      </property>
      <layout class="QHBoxLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lblCurrent">
         <property name="text">
          <string>Current State :</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="grpControl">
      <property name="title">
       <string>Control</string>
      </property>
      <layout class="QGridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="horizontalSpacing">
        <number>8</number>
       </property>
       <property name="verticalSpacing">
        <number>4</number>
       </property>
       <item row="3" column="4">
        <widget class="QPushButton" name="btnTerminate">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Terminate the DAQ session</string>
         </property>
         <property name="text">
          <string>Terminate</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblNextRunNumber">
         <property name="text">
          <string>Next RunN: </string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="btnInit">
         <property name="text">
          <string>Init</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QProgressBar" name="progressBar_scan">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QPushButton" name="btnStop">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Stop a run</string>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item row="5" column="4">
        <widget class="QPushButton" name="btnStartScan">
         <property name="text">
          <string>Start Scan</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblConfig">
         <property name="text">
          <string>Config file: </string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLineEdit" name="txtScanFile">
         <property name="text">
          <string>scan config file not set</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>ScanFile</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblInit">
         <property name="text">
          <string>Init file:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="txtInitFileName">
         <property name="text">
          <string>initialisation file not set</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="btnLoadInit">
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="btnLoadConf">
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="txtConfigFileName">
         <property name="whatsThis">
          <string extracomment="Configuration file - directly editable or through file dialog."/>
         </property>
         <property name="text">
          <string>configuration file not set</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="btnConfig">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Configure using the selected configuration</string>
         </property>
         <property name="text">
          <string>Config</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="btnStart">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Start a run</string>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="btnReset">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Reset all the DAQ clients</string>
         </property>
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="txtNextRunNumber"/>
       </item>
       <item row="5" column="3">
        <widget class="QPushButton" name="btn_LoadScanFile">
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLineEdit" name="txtLogmsg">
         <property name="toolTip">
          <string>Send a message to the LogCollector</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblLogmsg">
         <property name="text">
          <string>Log: </string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QPushButton" name="btnLog">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Send a log message</string>
         </property>
         <property name="text">
          <string>Log</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="grpStatus">
      <layout class="QGridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="horizontalSpacing">
        <number>8</number>
       </property>
       <property name="verticalSpacing">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <layout class="QGridLayout" name="grpGrid"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="grpConnections">
      <property name="title">
       <string>Connections</string>
      </property>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTreeView" name="viewConn">
         <property name="toolTip">
          <string>Processes connected to the Run Control</string>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1033</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>txtLogmsg</sender>
   <signal>returnPressed()</signal>
   <receiver>btnLog</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
